rel-eng/openshift-sdn-master.sysconfig

```diff
@@ -0,0 +1,44 @@
+# The $OPTIONS variable can specify any of the command-line options
+# available for openshift-sdn:
+#
+#  -alsologtostderr=false: log to standard error as well as files
+#  -etcd-cafile="": SSL Certificate Authority file used to secure etcd
+#    communication
+#  -etcd-certfile="": SSL certification file used to secure etcd
+#    communication
+#  -etcd-endpoints="http://127.0.0.1:4001": a comma-delimited list of
+#    etcd endpoints
+#  -etcd-keyfile="": SSL key file used to secure etcd communication
+#  -etcd-path="/registry/sdn/": etcd path
+#  -help=false: print this message
+#  -hostname="": Hostname as registered with master (for node mode),
+#    will default to 'hostname -f'
+#  -log_backtrace_at=:0: when logging hits line file:N, emit a stack
+#    trace
+#  -log_dir="": If non-empty, write log files in this directory
+#  -logtostderr=false: log to standard error instead of files
+#  -master=true: Run in master mode
+#  -minion=false: Run in minion mode
+#  -public-ip="": Publicly reachable IP address of this host (for node
+#   mode).
+#  -skip-setup=false: Skip the setup when in minion mode
+#  -stderrthreshold=0: logs at or above this threshold go to stderr
+#  -sync=false: Sync the minions directly to etcd-path (Do not wait
+#    for PaaS to do so!)
+#  -v=0: log level for V logs
+#  -vmodule=: comma-separated list of pattern=N settings for
+#    file-filtered logging
+#
+#   NOTE: values should not be quoted
+#
+# openshift-sdn runs in master mode by default, with any relevant
+# options set to the defaults documented above.
+#
+# Example:
+#   OPTIONS="-etcd-keyfile=/path/to/keyfile"
+OPTIONS=
+
+# The $DOCKER_OPTIONS variable is used to overwrite the $OPTIONS
+# variable in the /etc/sysconfig/docker file. This is a pretty ugly
+# hack, but whatevs
+DOCKER_OPTIONS='-b=lbr0 --mtu=1450 --selinux-enabled'
```

Comment:
We do not need to set any docker options for the master, do we?
We probably do not even require docker as a package for the master.